script.js

```diff
@@ -1,4 +1,480 @@
+// Scorekeeping Global States
+var wins = 0;
+var losses = 0;
+var draws = 0;
+var totalRounds = 0;
+var userWinPercentage = 0;
+var computerWinPercentage = 0;
+var winsInARow = 0;
+var lossesInARow = 0;
+var userLastWon = 0;
+var comLastWon = 0;
+
+// Other Global States
+var userName = "no user name";
+var reversedModeHint = 0;
+var gameMode = "waiting for user name";
+var computerHelp = 0;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "";
+  if (gameMode == "waiting for user name") {
+    userName = input;
+    gameMode = "waiting for game mode";
+    return `Nice to meet you <b>${userName}</b>, welcome to Scissors, Paper, Stone.<br>To begin, enter either:<br><br>1. Regular SPS<br>2. Korean SPS<br>3. Guess the Word<br>4. Dice Game</b>`;
+  }
+  // Select Game Modes (Korean SPS, Regular SPS or Guess the Word)
+  else if (input == "Korean SPS") {
+    gameMode = "Korean SPS";
+    return `Welcome to Korean Scissors, Paper, Stone. To begin, enter either 'scissors', 'paper' or 'stone'.<br><br><b><i>Additional Korean Version Rules: You only win a round if you win a match and follow-up with a draw in the next match.</b></i>`;
+  } else if (input == "Regular SPS") {
+    gameMode = "SPS";
+    return `Welcome to Regular Scissors, Paper Stone. To begin, enter either 'scissors', 'paper' or 'stone'.`;
+  } else if (input == "Guess the Word") {
+    gameMode = "Guess the Word";
+    return `Welcome to Guess the Word. To begin, enter the secret word.`;
+  } else if (input == "Dice Game") {
+    gameMode = "Dice Game";
+    return `Welcome to Dice Game. To begin, enter a number from 1 to 6.`;
+  }
+  // Error Message for either Korean SPS, Regular SPS, Guess the Word or Dice Game
+  else if (gameMode == "waiting for game mode") {
+    return `Please select a game mode before proceeding.<br>To begin, enter either 'Regular SPS', 'Korean SPS', 'Guess the Word' or 'Dice Game'.`;
+  }
+  // Activate Computer vs. Computer
+  else if (input == "Activate Lazy Mode" && computerHelp == 0) {
+    computerHelp = 1;
+    return `Computer will now help you choose.`;
+  }
+  // Deactivate Computer vs. Computer
+  else if (input == "Deactivate Lazy Mode" && computerHelp == 1) {
+    computerHelp = 0;
```

Comment:
noted your use of 0 and 1 for this, you can try using boolean values here
isComHelp = false